doc/howtos/building-images.rst

```diff
@@ -256,6 +284,51 @@ reconfigure it so that it matches ``ostro-image-swupd-dev``::
     OSTRO_IMAGE_NOSWUPD_EXTRA_FEATURES_append = "${OSTRO_IMAGE_FEATURES_DEV}"
     OSTRO_IMAGE_NOSWUPD_EXTRA_INSTALL_append = "${OSTRO_IMAGE_INSTALL_DEV}"
 
+Adding a Custom Layer in Ostro OS
```

Comment:
@gvancuts Geoffroy recommend simplifying this section for now.  Can you verify I've got this right @mythi